packages/components/Item/Types/components/ItemHeader.js

```diff
@@ -185,13 +186,13 @@ const generateTeamOptions = (teams, currentTeamId) =>
     .filter(({ id }) => id !== currentTeamId)
     .map(team => ({
       value: team.id,
-      label: team.id === 'personal' ? 'personal' : team.name,
+      label: team.id === TEAM_TYPE.PERSONAL ? TEAM_TYPE.PERSONAL : team.name,
```

Comment:
`team.id === TEAM_TYPE.PERSONAL` this condition will be put in const with redesign this component